tests/posix/getentropy/src/main.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2024 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/ztest.h>
+
+#include <unistd.h>
+
+ZTEST(getentropy_test_suite, test_getentropy_too_large)
+{
+	uint8_t buf[256 + 1] = { 0 };
+
+	int ret = getentropy(buf, sizeof(buf));
```

Comment:
You will need to have 1 empty line after the end of var declaration block

Reply:
Fixed. Thanks!